src/content/faq/General.md

```diff
@@ -9,70 +9,71 @@ questions: Why should I use GraphQL?,Is GraphQL a database language like SQL?,Do
+GraphQL clients can help you handle [queries and mutations](/learn/queries/) to a [GraphQL server](https://www.howtographql.com/advanced/1-server/). This could be done manually, but it would require a lot of work depending on the amount of data you have. By utilizing the underlying structure of a GraphQL API, clients can abstract away some of these tedious processes. Features that can be automated include batching, caching, UI updates, build-time schema validation, and many more. 
 
-## What is the GraphQL Foundation?
+You can find a list of GraphQL clients in various languages on our [Code page](/code/#graphql-clients). There’s also an [in-depth explanation of their benefits](https://www.howtographql.com/advanced/0-clients/) on How To GraphQL.
 
-<!-- TODO -->
+It takes more time to implement a GraphQL client upfront, but it becomes worth it the more features you need. You might want to start out [using HTTP requests](/learn/serving-over-http/) as the underlying transport layer (here’s [an example in JavaScript](/graphql-js/graphql-clients/)), then switch to a more intricate client service as your application grows in complexity.
 
-[Chupa chups gummies I love.]()
+## What is the GraphQL Foundation?
 
-Gummies marzipan dessert lemon drops muffin. Chocolate chocolate bar fruitcake toffee powder I love gummies. Donut tart cheesecake. Lollipop sweet roll apple pie I love. Pie candy canes jelly macaroon. Biscuit sugar plum chocolate bar cupcake pudding cake sugar plum bear claw. Cheesecake chocolate I love jelly beans I love tiramisu. Chocolate cake biscuit caramels halvah cotton candy pie. Gingerbread powder candy carrot cake jelly I love. Dessert I love chupa chups caramels lollipop candy canes gingerbread I love gummi bears. 
+The [GraphQL Foundation](https://foundation.graphql.org/faq/) provides governance for GraphQL as well as vendor-neutral oversight of funding, events, operations resources, and more. It was formed in 2018 by [various tech companies](https://landscape.graphql.org/category=graph-ql-foundation-member&format=logo-mode) and hosted under the [Linux Foundation](https://www.linuxfoundation.org/). The idea is that it’s an open, neutral home for the GraphQL community.
 
-Fruitcake lollipop danish cheesecake liquorice cookie cheesecake toffee oat cake.
+You can find out more by visiting [foundation.graphql.org](https://foundation.graphql.org/).
 
 ## How can I contribute to the GraphQL specification?
 
-<!-- TODO -->
-
-Carrot cake tootsie roll ice cream candy canes bonbon. Pudding macaroon I love danish dragée tart biscuit. [Biscuit cake wafer.]()
\ No newline at end of file
+GraphQL is still an evolving language and contributions are very welcome! The specification (including the [latest working draft](https://spec.graphql.org/)) is open source and [contributor guidelines](https://github.com/graphql/graphql-spec/blob/master/CONTRIBUTING.md) are available on GitHub.
```

Comment:
should this question be expanded into also open source libraries together with the spec?

Reply:
This was a debate I had early-on as well - should this be a general contributing question or specific for the spec. I went with the spec because it was proposed in the original faq repo, but I don't see the harm in expanding it (as long as the answer doesn't become long and bloated). 

Any specific libraries you think should be highlighted? Or do you mean more of a general 'we also have some libraries' note?

Reply:
I guess we could mention the foundation libraries like graphql-js, GraphiQL express-graphql?
and also maybe link to the code section?
also maybe mentioning contributing to the website itself!